WhatBackend.TelegramBotApi/Startup.cs

```diff
@@ -39,14 +37,15 @@ public void ConfigureServices(IServiceCollection services)
                   b => b.MigrationsAssembly("CharlieBackend.TelegramBotApi"));
             });
 
+            CompositionRoot.InjectDependencies(services, Configuration);
```

Comment:
Why do you decided to inject all the dependencies from WHAT project if you don't need it?